pymemcache/client/base.py

```diff
@@ -109,6 +111,28 @@ def _check_key(key, allow_unicode_keys, key_prefix=b''):
     return key
 
 
+class _WrappingSerde(object):
+    def __init__(self, serializer_func, deserializer_func):
+        self.serializer = serializer_func
+        self.deserializer = deserializer_func
+
+
+def _load_serde(serde, serializer, deserializer):
+    """helper for handling compatible "serde" object loading"""
+    if serde:
+        return serde
+    elif serializer and deserializer:
+        warnings.warn(
+            "pymemcache.base.Client support for the serializer and "
+            "deserializer functions is deprecated and will be removed in "
+            "a future version of pymemcache. Use the 'serde' object instead.",
```

Comment:
I don't want to commit to saying will be removed in a future version I would rather leave it as deprecated but have no plans to remove it unless we need to. I don't want to force extra work for existing users if possible